### DISCUSSION TITLE
Edge Functions Deploy Error

### DISCUSSION THREAD

**Discussion by restaurantt** [supabase/supabase#12820] [Questions]
I am trying to deploy openai api on edge functions
but when I deploy it with the command "supabase functions deploy OpenAI",  here it the error log, I don't know what I have missed from the tutorial,
Can any one point where am I wrong?

> Version 1.30.3 is already installed
Bundling OpenAI
Error: Error bundling function: signal: trace/BPT trap
file:///src/import_map.json
file:///src/index.ts
assertion failed [block != nullptr]: BasicBlock requested for unrecognized address
(BuilderBase.h:550 block_for_offset)

Try rerunning the command with --debug to troubleshoot the error.

and follow is my code:

`import { xhr } from 'https://deno.land/x/xhr@0.3.0/mod.ts';
import { serve } from 'https://deno.land/std@0.177.0/http/server.ts';
import { Configuration, OpenAIApi, CreateCompletionRequest } from 'https://esm.sh/openai@3.1.0';

const configuration = new Configuration({
  apiKey: 'MyApiKey',
});

const openai = new OpenAIApi(configuration);

serve(async (req) => {
  const { model, prompt, max_tokens, temperature } = await req.json();

  const completionConfig: CreateCompletionRequest = {
    model: model,
    prompt: prompt,
    maxTokens: max_tokens,
    temperature: temperature,
    stream: true,
  };

  const headers = {
    Authorization: `Bearer ${configuration.apiKey}`,
    'Content-Type': 'application/json',
  };

  const { response } = await xhr({
    url: 'https://api.openai.com/v1/completions',
    method: 'POST',
    headers: headers,
    body: JSON.stringify(completionConfig),
  });

  return new Response(response);

  // return new Response(response.body, {
  //   headers: {
  //     'Content-Type': 'application/json',
  //   },
  // });

});`

**Comment by oliverbytes** [comment]
Any solution?
